blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/IOConfig.java

```diff
@@ -124,6 +124,13 @@ public int getRequestTimeout() {
         return config.getIntegerProperty(CoreConfig.CASSANDRA_REQUEST_TIMEOUT);
     }
 
+    public int getReadTimeoutMaxRetries() { return config.getIntegerProperty(CoreConfig.DATASTAX_READ_TIMEOUT_MAX_RETRIES); }
```

Comment:
Because it's related to how we configure IO/driver? The other configuration, such as CASSANDRA_REQUEST_TIMEOUT, etc , are fetched from here as well. I thought I put them all in same place.